operator-common/src/main/java/io/strimzi/operator/common/Annotations.java

```diff
@@ -150,6 +150,11 @@ public class Annotations {
      */
     public static final String ANNO_DEP_KUBE_IO_REVISION = "deployment.kubernetes.io/revision";
 
+    /**
+     * Annotation for enabling/disabling usage of KafkaNodePools in Kafka CR
+     */
+    public static final String ANNO_STRIMZI_IO_NODE_POOLS = STRIMZI_DOMAIN + "node-pools";
```

Comment:
You should not add a new annotation. Assuming you need it in the `operator-common` module, you should move here the existing annotation.